source/common/config/filter_json.cc

```diff
@@ -262,6 +262,8 @@ void FilterJson::translateFaultFilter(
 
   const Json::ObjectSharedPtr json_config_abort = json_config.getObject("abort", true);
   const Json::ObjectSharedPtr json_config_delay = json_config.getObject("delay", true);
+  const Json::ObjectSharedPtr json_config_abort_runtime = json_config.getObject("abort_runtime", true);
```

Comment:
Revert all changes to this file. We no longer make any v1 config changes. Same in the schema file below.